install-from-binstall-release.ps1

```diff
@@ -0,0 +1,16 @@
+$tmpdir = $Env:TEMP
+$type = (Get-ComputerInfo).CsSystemType
+if ($type.StartsWith("x64")) {
+	$url = "https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-pc-windows-msvc.zip"
```

Comment:
Can we have a base_url, similar to the unix version?

Reply:
I've updated the script to use a `$base_url` variable